collector/qdisc_linux.go

```diff
@@ -0,0 +1,85 @@
+// Copyright 2017 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build !noqdisc
+
+package collector
+
+import (
+	"github.com/ema/qdisc"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+type qdiscStatCollector struct {
+	bytes      typedDesc
+	packets    typedDesc
+	drops      typedDesc
+	requeues   typedDesc
+	overlimits typedDesc
+}
+
+func init() {
+	Factories["qdisc"] = NewQdiscStatCollector
+}
+
+func NewQdiscStatCollector() (Collector, error) {
+	return &qdiscStatCollector{
+		bytes: typedDesc{prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, "qdisc", "bytes_total"),
+			"Number of bytes sent.",
+			[]string{"iface", "kind"}, nil,
```

Comment:
Looks like most network collectors use the label "device" instead of "iface".  Can I get a confirmation from maintainers?